apps/script/HoC2023ScriptFiles/README.md

```diff
@@ -18,116 +18,18 @@ The script generates/updates 3 files in apps/static/dance/ai/model:
 - cached_palettes_map.json
 
 These files store association values measuring the similarity between an emoji and each output using embeddings generated by a LLM.
-An embedding is a vector (list) of floating point numbers, and embeddings are used to
-measure the relatedness of text strings. The distance between two vectors measures
-their relatedness. https://developers.google.com/machine-learning/crash-course/embeddings/video-lecture
-These embeddings are stored in caches files as pickle files, python's native way to serialize data.
+The embeddings used to generate these maps are stored in cached pickle files such as foreground_embeddings.pkl to prevent duplicate LLM API calls.
 
-## Architecture and Background
+## Expected Runtime Behavior
 
-At runtime, DanceAI will use the three maps to lookup the scores for each output type and take the top three indexes of (MAX(SUM(Input1Scores, Input2Scores, Input3Scores))) to select a final palette/foreground/background to display to the user. These maps are stored as a local cache rather than generated at runtime to remove the costs associated with querying a LLM and improve runtime performance.
+At runtime, DanceAI will use the three maps to lookup the scores for each output type and randomly select one of the top 3 results of MAX(SUM(Input1Scores, Input2Scores, Input3Scores)) to select a final palette/foreground/background to display to the user. These maps are stored as a local cache rather than generated at runtime to remove the costs associated with querying a LLM and improve runtime performance.
```

Comment:
Since on the frontend, we present both randomly selected top version (from top 3) and randomly selected bottom versions ( 4 from bottom 20), maybe we can just state that the three maps are used to look up scores for each output type. We could also refer them to the `calculateOutputSummedWeights.ts` file.